src/collections/_documentation/platforms/cocoa/usage.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Advanced Usage'
+---
+
+## Capturing uncaught exceptions on macOS
+
+By default macOS application do not crash whenever an uncaught exception occurs. There are some additional steps you have to take to make this work with Sentry. You have to open the applications `Info.plist` file and look for `Principal class`. The content of it which should be `NSApplication` should be replaced with `SentryCrashExceptionApplication`.
+
+## Dealing with Scopes
+
+There are four different ways to interact with Scopes in the SDK.
+
+### configureScope
+
+Using `SentrySDK:configureScope` lets you set context data globally, this will be attached to all future events.
+
+{% include components/platform_content.html content_dir='configureScope' %}
+
+### Passing a Scope Instance
+
+By passing an instance of a Scope directly, it will not merge with the current global Scope. 
+This is helpful in cases where you completly want to control what should be attache to the event.
```

Comment:
Suggest "Setting an instance of Scope is helpful when you want to completely control what should be attached to the event."